docs/src/developers_guide/contributing_documentation.rst

```diff
@@ -24,6 +24,9 @@ The documentation uses specific packages that need to be present.  Please see
 Building
 ~~~~~~~~
 
+This documentation is built using the latest Python version that Iris
```

Comment:
@tkknight Could we change to tense here from `documentation is built` to `documentation was built`

Reply:
Done.  See 793a219